### PR TITLE
feat(args): use newly added multiline variable reader

### PR DESCRIPTION
**Pull request by k-dovgan**

### CODE REVIEW

**Comment by k-dovgan** [comment]
As an alternative 'read_multiline_variable' can be united with 'check_required_option' and accept settings as parameter and raise 'empty var!' exception itself.

File: universum/github_handler.py [hunk 1]
```diff
@@ -59,16 +52,26 @@ def __init__(self, *args, **kwargs):
                     Please specify this parameter by using '--trigger-url' ('-tu')
                     command line parameter or by setting TRIGGER_URL environment variable.
                 """)
+        try:
+            self.payload = utils.read_multiline_option(self.settings.payload)  # actually may throw AttributeError
+            if not self.payload:
+                raise AttributeError("Empty payload")
+        except AttributeError:
+            raise IncorrectParameterError("""
```
Comment (i-keliukh) [inline universum/github_handler.py:60]: cleandoc is missing

File: universum/github_handler.py [hunk 2]
```diff
@@ -59,16 +52,26 @@ def __init__(self, *args, **kwargs):
                     Please specify this parameter by using '--trigger-url' ('-tu')
                     command line parameter or by setting TRIGGER_URL environment variable.
                 """)
+        try:
```
Comment (i-keliukh) [inline universum/github_handler.py:55]: This whole block should be moved to utils as well.

Comment (k-dovgan) [reply universum/github_handler.py:55]: that's exactly what I've written in comment above. The question is: should it become a part of 'read' function? Should the function become 'read_and_check' or two separate functions?

File: universum/lib/utils.py [hunk 3]
```diff
@@ -93,21 +93,21 @@ def check_required_option(settings, setting_name, error_message):
...
+                with open(value.lstrip('@')) as file_name:
+                    result = file_name.read()
+            except FileNotFoundError as e:
+                raise IncorrectParameterError(f"Error reading argument {setting_name} from file {e.filename}: no such file")
+        elif value == '-':
+            result = sys.stdin.read()
+        else:
+            result = value
+        if not result:
+            raise AttributeError('empty value')
```
Comment (i-keliukh) [inline universum/lib/utils.py:107]: I think raising IncorrectParameterError has the benefit of keeping original stack trace and is cleaner.

**Pull request merged**